src/ets/scala/io/iohk/ethereum/ets/blockchain/BlockchainTestConfig.scala

```diff
@@ -235,15 +235,15 @@ object Eip158ToByzantiumAt5Config extends BlockchainTestConfig {
     MonetaryPolicyConfig(5000000, 0.2, BigInt("5000000000000000000"), BigInt("3000000000000000000"))
 }
 
-object Validators {
-  val frontierValidators = EthashValidators(FrontierConfig)
-  val homesteadValidators = EthashValidators(HomesteadConfig)
-  val eip150Validators = EthashValidators(Eip150Config)
-  val frontierToHomesteadValidators = EthashValidators(FrontierToHomesteadAt5)
-  val homesteadToEipValidators = EthashValidators(HomesteadToEIP150At5)
-  val homesteadToDaoValidators = EthashValidators(HomesteadToDaoAt5)
-  val eip158Validators = EthashValidators(Eip158Config)
-  val byzantiumValidators = EthashValidators(ByzantiumConfig)
-  val constantinopleValidators = EthashValidators(ConstantinopleConfig)
-  val eip158ToByzantiumValidators = EthashValidators(Eip158ToByzantiumAt5Config)
+case class Validators(shouldSkipPoW: Boolean) {
```

Comment:
As i rembemer i was all created in object, because otherwise pow cache would need to be recreated before each test which leads to really long running times for ets test. What is ets time after this change ? (Maybe it did not changed much because of this new field, which disabled pow for most of tests)

Reply:
Good question. Still, if the validators will be a `def`s they will have to be recreated.

Reply:
this branch: 19m 45s 162ms
constantinople branch: 36s 295ms

so I'll change it ti previous one

Reply:
To which one? If you change to defs inside an object it won't change anything (I guess) because it will be evaluated each time

Reply:
with defs it is taking 18m 24s 488ms. I tried also with trait and only `shouldSkipPoW` as a def but it takes: 18m 5s 836ms.

I've done something like @KonradStaniec mention in his previous comment and now tests run in: 9m 1s 652ms while I check